recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ source:
 build:
   number: {{ build_num }}
   # Package not supported on win32 and s390x
-  skip: true  # [win32 or s390x or (aarch64 and py>39)]
+  skip: true  # [not (osx and arm64)]
```

Comment:
Why is this only built for `osx-arm64`? Could you specify in the comments?

Reply:
@marcoesters Sorry, I requested my review _before_ it was quite ready. Building with Prefect revealed a packaging defect that I was not able to recreate elsewhere, so I had to do it in a PR and push to `c3i_test2` via an `abs.yaml` entry. This is in place to reduce the impact on our build system since we are only really interested in the `osx-arm64` package.

So you are seeing artifacts of the process necessary to generate that build here. When I rerequest reviews, I will revert to the prior hash.

All that should be in this PR is a small change in the CMake options to disable the python2 search (and resulting `python2.7/site-packages` path) and increment of the build number.